src/DonationForms/Blocks/DonationFormBlock/resources/editor/hooks/useFormOptions.ts

```diff
@@ -31,5 +30,6 @@ export default function useFormOptions(): {formOptions: Option[] | []; isResolvi
     return {
         isResolving,
         formOptions,
+        forms,
```

Comment:
There is a ton of data loaded here, but we use only a few properties. Do we really need all that? If we need only `formTemplate` for helper functions like `isLegacyForm`, we can add that property into `formOptions`. We really don't need all the stuff that `getEntityRecords` returns.

Reply:
Resolved: https://github.com/impress-org/givewp/pull/7082/commits/b70178c7ee9134c7bd498b97aad17656b3ca830f